docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -117,7 +117,8 @@ The file represents a collection of JSON elements describing known issues.
             "var":  "value-.*"
         },
         "failScenarioFast": true, <8>
-        "failStoryFast": true <9>
+        "failStoryFast": true, <9>
+        "description": "Missing listener on a button" <10>
```

Comment:
it's better to have example which matches the issue, in general issue describes some REST API issue:
```suggestion
        "description": "Missing endpoint configuration on server side" <10>
```